terraform/test/main.tf

```diff
@@ -40,4 +40,24 @@ module "active_directory" {
   number_of_domain_controllers = 2
   ldaps_ca_subnet              = module.networking.ldaps_ca_subnet
   environment                  = "test"
-}
\ No newline at end of file
+}
+
+resource "aws_key_pair" "bastion_ssh_key" {
+  key_name   = "bastion_ssh_key"
```

Comment:
Also this needs to include environment in the name to avoid clashing with staging.